moped-database/config/primary_key_map.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This file needs to be placed in S3 in a particular location, with one copy that reflects the current schema per deployment stage. Right now, we have one for `test`, `staging` and `production`.